packages/tupaia-web/src/features/Map/MapOverlaySelector/DesktopMapOverlaySelector.tsx

```diff
@@ -49,6 +49,7 @@ const Heading = styled(Typography).attrs({
 
 const Container = styled(MaxHeightContainer)`
   border-radius: 0 0 5px 5px;
+  pointer-events: auto;
```

Comment:
This is super pedantic, but I wonder if `pointer-events: auto` should go in `MaxHeightContainer` since it's used as the base for both `Wrapper` and `Container` (assuming it isn't used anywhere else in the file)

Reply:
I actually had the pointer events set on the wrapper initially but it was blocking map interactions when the menu was closed so I moved it down. I've added a comment.